tests/SprykerEcoTest/Zed/Computop/Business/Api/AbstractPaymentTest.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEcoTest\Zed\Computop\Business\Api;
+
+use GuzzleHttp\Psr7;
+use Orm\Zed\Computop\Persistence\SpyPaymentComputop;
+use Orm\Zed\Computop\Persistence\SpyPaymentComputopDetail;
+use Orm\Zed\Computop\Persistence\SpyPaymentComputopOrderItem;
+use SprykerEco\Service\Computop\ComputopService;
+use SprykerEco\Shared\Computop\ComputopConfig;
+use SprykerEco\Shared\Computop\Config\ComputopFieldName;
+use SprykerEco\Zed\Computop\Business\Api\ComputopBusinessApiFactory;
+use SprykerEco\Zed\Computop\Business\ComputopBusinessFactory;
+use SprykerEco\Zed\Computop\ComputopConfig as SprykerComputopConfig;
```

Comment:
Wrong alias name — I would suggest `ZedComputopConfig` (or the other can be aliased as `SharedComputopConfig` — you choose)